libs/domain/content/link/link.model.ts

```diff
@@ -5,6 +5,12 @@ export interface ContentLinkContent {
   text?: string;
 }
 
+export const enum ContentLinkAppearance {
+  Link = 'link',
+  Dropdown = 'dropdown',
```

Comment:
I'd prefer to try to make this `Navigation`, so we can use it in consistently in all navigations, such as the category navigation and the in-page my account navigation. It would mean that the style rules are generic enough. If we cannot do it like this, we can stick with Dropdown. 
The problem with making it `dropdown` is that we cannot easily move an navigation from one composition to another without changing the appearance on the components. We'd also not be able to reuse it. 

We'll also tackle this with UX to see if we can make those navigations fairly consistent from style rules perspective.